recipes/pyxtensor-feedstock/build.bat

```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles" -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR%
```

Comment:
Please use `Ninja` or `NMake Makefiles JOM` for parallelism.

Reply:
Do I have to specify the number of CPUs also with `nmake` (like with `make` for unix-based systems)?